tests/test_abstract_repr.py

```diff
@@ -1050,13 +1075,27 @@ def _get_expression(op: dict) -> Any:
 
 
 class TestDeserialization:
```

Comment:
I see a lot of warning catches in this class that I guess we don't really care for. What if we use something like this to avoid them? https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-mark-filterwarnings

Reply:
Okay I hesitated with using this, in the mean time I found it nice to know which function raises a warning and which does not, but now that we know nothing is weird I guess I can use filterwarnings